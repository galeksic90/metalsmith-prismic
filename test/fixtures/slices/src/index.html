---
template: index.hbt
prismic:
  slices:
    query: '[[:d = any(document.type, ["slices"])]]'
  slices-group:
    query: '[[:d = any(document.type, ["slices-group"])]]'
---
