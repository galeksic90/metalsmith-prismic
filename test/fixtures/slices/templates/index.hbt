<html>
<body>
<ul>
{{#each prismic.slices.results}}
    <li>
        {{#each data.body.children}}
        <div class="{{this.sliceType}}">
            {{{this.html}}}
        </div>
        {{/each}}
    </li>
{{/each}}
</ul>
<ul>
{{#each prismic.slices-group.results}}
    <li>
        {{#each data.body.children}}
        <div class="{{this.sliceType}}">
            {{!-- {{Workaround for missing switch-case in Handlebars}} --}}
            {{#each this.children}}
            <div class="image">
                {{{this.illustration.html}}}
                {{{this.caption.html}}}
            </div>
            {{else}}
            {{{this.html}}}
            {{/each}}
        </div>
        {{/each}}
    </li>
{{/each}}
</ul>
</body>
</html>
