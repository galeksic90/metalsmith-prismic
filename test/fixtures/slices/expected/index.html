<html>
<body>
<ul>
    <li>
        <div class="image">
            <img src="https://prismic-io.s3.amazonaws.com/api-test/0d5f11ae169105248144293669bd526369ca3a23_jean-michel.jpg" width="640" height="960" alt="">
        </div>
        <div class="text">
            <p>Jean-Michel Pastranova, the founder of <em>Les Bonnes Choses</em>, and creator of the whole concept of modern fine pastry, has decided to step down as the CEO and the Director of Workshops of <em>Les Bonnes Choses</em>, to focus on other projects, among which his now best-selling pastry cook books, but also to take on a primary role in a culinary television show to be announced later this year.</p>
        </div>
        <div class="quote">
            <p>I believe I've taken the Les Bonnes Choses concept as far as it can go. Les Bonnes Choses is already an entity that is driven by its people, thanks to a strong internal culture, so I don't feel like they need me as much as they used to. I'm sure they are greater ways to come, to innovate in pastry, and I'm sure Les Bonnes Choses's coming innovation will be even more mind-blowing than if I had stayed longer.</p>
        </div>
        <div class="text">
            <p>He will remain as a senior advisor to the board, and to the workshop artists, as his daughter Selena, who has been working with him for several years, will fulfill the CEO role from now on.</p>
        </div>
        <div class="quote">
            <p>My father was able not only to create a revolutionary concept, but also a company culture that puts everyone in charge of driving the company's innovation and quality. That gives us years, maybe decades of revolutionary ideas to come, and there's still a long, wonderful path to walk in the fine pastry world.</p>
        </div>
    </li>
</ul>
<ul>
    <li>
        <div class="gallery">
            <div class="image">
                <img src="https://prismic-io.s3.amazonaws.com/api-test/2829e3badec6a5a8c6c4a4ea045f0c0f4499a7ec_pastry-art-1.jpg" width="900" height="900" alt="">
                <h3>Preparation</h3>
            </div>
            <div class="image">
                <img src="https://prismic-io.s3.amazonaws.com/api-test/dd75e6a7ad4255d4e72091682f1ea860cc42b737_pastry-art-2.png" width="900" height="420" alt="">
                <h3>Teamwork</h3>
            </div>
            <div class="image">
                <img src="https://prismic-io.s3.amazonaws.com/api-test/71a71eeba3b79ed242d09308f62887cee8891444_pastry-art-3.jpg" width="900" height="600" alt="">
                <h3>Satisfied tablet user!</h3>
            </div>
        </div>
        <div class="text">
            <p>Each year, Les Bonnes Choses organizes a world-famous two-day event called the "Pastry Art Brainstorm", and which is the perfect excuse for every fine pastry artist in the world to exercise their art, and build their skills up. The event is a multiple win-win operation, at many levels: see what the event is, as seen by many point of views.</p><h2>As seen by the top pastry artists worldwide</h2><p>The event always starts with half a day of conference talks, given by the most insightful pastry artists in the world, selected for having made tremendous achievements in pastry that year. The list of invited guest speakers is decided jointly by the Les Bonnes Choses staff and the Fine Pastry Magazine editors.</p><p>This is great for the speakers, who get an occasion to share their work, and have people build up on it with them.</p><h2>As seen by the pastry professionals</h2><p>After half a day of thoughtful conference, the professionals will get to put what they learned to good use, and mingle with the best artists worldwide to make the most daring pastries together. There are no set rules about who does what during this giant innovation workshop, and many crazy ideas get created out of thin air. As a virtually infinite amount of ingredients is provided by the Les Bonnes Choses staff, many unexpected pastries happen on that day, and professionals taste each other's creations, and provide relevant feedback to each other. Most pieces get showcased to the amateur audience as well, who get invited to taste some of the pieces.</p><p>At noon on the second day, teams are expected to subscribe to our Pastry Art Challenge, during which they will make the best possible pastry, judged on many aspects (originality, taste, looks, ...) by a jury of amateurs and professionals. The team members of the three winning pieces share a substantial prize, and their pastries may even join the Les Bonnes Choses catalogue, and be offered in all the Les Bonnes Choses shops worldwide!</p><h2>As seen by the pastry amateurs</h2><p>The conference is limited with a reasonable fee; but the showcase is open to everyone, although visitors are often expected to pay the pastry chefs for the pastries they taste. The educated amateurs spend their day tasting the most daring pieces, giving some appreciated feedback to their chefs, and challenging their own tastebuds. The novice amateurs usually get a once-in-a-lifetime experience, and often mention being blown away by how rich the fine pastry art can be. All in all, every one goes home with a smile on their faces!</p><h2>As seen by the Les Bonnes Choses interns</h2><p>Every year, we recruit a very limited amount of interns, who get aboard a life-defining adventure around fine pastries, discovering Les Bonnes Choses during half a year, with part of this time spent in one of our shops abroad. We always manage to get them on board at a time when we know they will be able to attend a Fine Pastry Brainstorm, because we consider it is a very defining element in the experience of being part of Les Bonnes Choses.</p><p>Not only do we invite them to the event (whatever the country they are stationed in when the event happens), but we give them a front-row seat! They are part of the jury for the Fine Pastry Challenge, they are introduced to every speaker as the next generation of pastry (thus having the occasion to learn even more, directly from them).</p><h2>As seen by fine pastry as a field</h2><p>There wasn't really an international occasion for pastry artists to join and share, before Les Bonnes Choses came up with the first Fine Pastry Brainstorm, in 2006. Fine Pastry Magazine's first edition was out in 2004, and initiated the idea that pastry art needed to be shared better between professionals. But a proper event to meet up in person was missing, and Les Bonnes Choses is proud to be the one to have come up with it first.</p><p>Since then, more local initiatives have been started (notably in Argentina, and Canada), but none comes close to the size of Les Bonnes Choses's international Fine Pastry Brainstorm.</p><h2>As seen by Les Bonnes Choses</h2><p>As the almost only sponsor of every edition of the event, Les Bonnes Choses makes sure enough ingredients are available for everyone, rents the premises, makes sure the speakers are as comfortable as possible, and takes care of the whole organization! But through the operation, Les Bonnes Choses gains much more than any sponsoring can buy: not only does it get to secure Les Bonnes Choses as the world reference in pastry arts, but it also allows them to claim rightfully that they do offer in their shops the best pastries, created by the world top artists indeed.</p>
        </div>
    </li>
</ul>
</body>
</html>
